src/main/java/org/apache/maven/plugin/surefire/report/TreePrinter.java

```diff
@@ -77,7 +79,7 @@ public void printTests() {
         testSetStats.getReportEntries()
                 .stream()
                 .map(TestPrinter::new)
-                .forEach(TestPrinter::printTest);
+                .forEach(testPrinter -> testPrinter.printTest(classResult));
```

Comment:
Return to Method Reference by using this parameter directly from the TreePrinter's field